mailbox/cassandra/pom.xml

```diff
@@ -156,6 +156,7 @@
         <dependency>
             <groupId>org.cassandraunit</groupId>
             <artifactId>cassandra-unit</artifactId>
+            <version>2.1.9.2</version>
```

Comment:
AFAIK we don't use cassandra-unit anymore

Reply:
damn, I didn't notice that 🙄